Feat: I updated the styles on the login form.

StudioWythe opened the pull request:
I updated button color and adjusted margin on the field items for the login form.

StudioWythe committed:
Feat: I updated the styles on the login form.

alexaverill commented:
This causes login form elements to be expanded outside of the window on small mobile screens.  Please ensure css styles conform to responsive design. 
![Screen Shot 2020-10-22 at 19 00 59](https://user-images.githubusercontent.com/790022/96944751-f3097580-1498-11eb-9369-967c332c551f.png)

StudioWythe committed:
Feat: Updated Fonts

I updated the fonts, mostly for fun, plus I fixed the responsive issue on the login form.

alexaverill reviewed (approved):
Looks pretty good.
